### ISSUE TITLE
Update documentation on ITK version

### ISSUE THREAD

**Issue by minghuaw** [OpenGATE/Gate#348]
The latest stable release of ITK is version 5.1.0, which is not compatible with GATEv9.0. I successfully compiled Gate with ITK-4.13.2, which is the previous stable release, and the ThermalActor can be successfully added in an interactive Gate session. Could you please update the documentation regarding the version requirement for ITK?

Thanks

**Comment by tbaudier** [comment]
Hi @minghuaw ,
Could you tell us more about the compilation error you have, please?

With the CI, we compile the latest commit of Gate with the latest commit of ITK (https://github.com/OpenGATE/Gate/blob/develop/azure-pipelines.yml#L182) and it works on [Linux](https://dev.azure.com/OpenGateCollaboration/Gate/_build/results?buildId=126&view=logs&j=c4202c95-8789-530d-75df-fc930dff3ff5) and [Mac](https://dev.azure.com/OpenGateCollaboration/Gate/_build/results?buildId=126&view=logs&j=f1f5d5a6-84b2-5efe-b69c-900ea5fc9dd2)

Thank you
Thomas

**Comment by minghuaw** [comment]
Thanks for you response. I have deleted my ITK-5.1.0 build after it failed in the Gate compilation, so I actually do not have any record of the error message. However, I just downloaded and compiled ITK-5.1.0 again, and use the new build to compile Gate, and fortunately everything works perfectly this time. I might have toggled some unnecessary options for the ITK-5.1.0 build the last time. I guess we can close the issue.

Cheers
MW
